modules/core/core-api/src/main/java/com/enonic/xp/node/NodeService.java

```diff
@@ -29,14 +29,18 @@
 
     Node getById( NodeId id );
 
+    @Deprecated
```

Comment:
These are in use in Content API, and should be redone, too.

Reply:
Maybe just keep these methods, without deprecation. But don't use NodeId internally - leave it for another round of refactoring